pkgs/development/python-modules/mypy-protobuf/default.nix

```
@@ -10,14 +10,14 @@
 
 buildPythonPackage rec {
   pname = "mypy-protobuf";
-  version = "3.4.0";
+  version = "3.5.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/nipunn1313/mypy-protobuf/blob/17c22141f3f45a181f5c310e929881bffb38146c/setup.cfg#L20